utils/m0reportbug

```diff
@@ -748,6 +749,16 @@ addb() {
              ## > lt-m0addb2dump: Cannot create stob: -2: No such file
              ## > or directory
              $m0addb2dump $path | tail -$ADDB_TAIL | _xz >$outdir$path.xz
+             dumpfile_path=$outdir$(dirname $path)
+             $m0addb2dump -f -- $path > ${dumpfile_path}/dump_1.txt
+             if [[ -f $addb2db ]]; then
```

Comment:
Please add double quote `"` around the variable name:
```
if [[ -f "$addb2db" ]]; then
```

Usually this would work, but if the variable name contains some special chars, it would not work as expected.